From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fgei <fgei@gmail.com>
Date: Wed, 14 May 2025 05:22:22 +0000
Subject: [PATCH] enable certificate transparency feature by default for
 browser

Adapted from GrapheneOS Vanadium patch 0204-enable-certificate-transparency-feature-by-default-f.patch for the Vandium browser port.

---
 chrome/browser/browser_features.cc | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/chrome/browser/browser_features.cc b/chrome/browser/browser_features.cc
index e181af9ee59b1..360105d596afe 100644
--- a/chrome/browser/browser_features.cc
+++ b/chrome/browser/browser_features.cc
@@ -54,11 +54,7 @@ BASE_FEATURE(kBookmarkTriggerForPreconnect,
 // switch.
 BASE_FEATURE(kCertificateTransparencyAskBeforeEnabling,
              "CertificateTransparencyAskBeforeEnabling",
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
              base::FEATURE_ENABLED_BY_DEFAULT);
-#else
-             base::FEATURE_DISABLED_BY_DEFAULT);
-#endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
 
 // Enables using network time for certificate verification. If enabled, network
 // time will be used to verify certificate validity, however certificates that
